cmd/helm/list.go

```diff
@@ -104,7 +104,7 @@ func newListCmd(client helm.Interface, out io.Writer) *cobra.Command {
 	f.BoolVarP(&list.sortDesc, "reverse", "r", false, "reverse the sort order")
 	f.IntVarP(&list.limit, "max", "m", 256, "maximum number of releases to fetch")
 	f.StringVarP(&list.offset, "offset", "o", "", "next release name in the list, used to offset from start value")
-	f.BoolVar(&list.all, "all", false, "show all releases, not just the ones marked DEPLOYED")
+	f.BoolVar(&list.all, "all", "a", false, "show all releases, not just the ones marked DEPLOYED")
```

Comment:
- [ ] I think you meant to use [`pflag.BoolVarP`](https://godoc.org/github.com/spf13/pflag#BoolVarP) here instead.